### ISSUE TITLE
Consider addding more information: license and tech stack

### ISSUE THREAD

**Issue by alexkreidler** [smknstd/privacy-focused-analytics#6]
The license and tech stack often are important to people who want to self host, as they determine whether they can do so for commercial projects and the complexity of hosting each solution.

Although this might increase the amount of work required to maintain this data, it could just be optional so we don't have to add it or keep it up to date for every project.

I've started collecting that data for a few projects [here](https://gist.github.com/alexkreidler/21499364a8c8e3ca1cca02dabdd6a822).

Also, I'm curious which of these analytics tools you use, if any. This list is really great but it gave me a bit of an overload from all the options. Maybe there would be a way to have recommended tools for various users: e.g. a beginner, a self-hoster, someone who wants a lot of raw data. I don't know, you may want to keep it as an impartial list.

**Comment by smknstd** [comment]
Great feedback ! I started this ist with only 7 services so it was a lot easier to figure out all available solutions. But the list keeps growing and it takes a lot more time to explore everything. I'd like to keep the list as partial as possible. But I don't have any idea how to improve UX right now. Github stars count would be great as seen on https://jamstack.org/generators/.

PS: Adding infos on tech stack would be great too.

**Comment by alexkreidler** [comment]
Yeah, as the list grows it gets more complicated. I think Github stars are a great idea.

For the UX, one idea could be to render it as table, with each tool's icon, name, and the rest of the properties. There might be some Vue components for that, but I'm not really familiar with Vue.
